### PR TITLE
Chore: a bit of spring cleaning

### PR DESCRIPTION
**Pull request by markelog**
This pull re-enables some of the revive rules and fixes associated "errors" with it.

It seems some of it is subjective, so I would say not set on stone :)

### CODE REVIEW

File: pkg/cmd/grafana-server/server.go [hunk 1]
```diff
@@ -10,38 +10,36 @@ import (
 	"path/filepath"
 	"strconv"
 	"time"
+	// self registering services
```
Comment (bergquist) [inline pkg/cmd/grafana-server/server.go]: This comment can be removed

**Review by bergquist — COMMENTED** [review]
Please remove the comment in server.go

Looks good otherwise.

File: pkg/cmd/grafana-server/server.go (hunk 1 above)
Comment (markelog) [reply pkg/cmd/grafana-server/server.go]: Oh yeah, I use a plugin for vscode now that changes sorting of imports

Comment (markelog) [reply pkg/cmd/grafana-server/server.go]: Done!

**Review by bergquist — APPROVED** [review]
LGTM

**Pull request merged**